src/Symfony/Component/Console/Application.php

```diff
@@ -360,7 +360,7 @@ public function add(Command $command)
         if (!$command->isEnabled()) {
             $command->setApplication(null);
 
-            return;
+            return null;
```

Comment:
then the docs or off;
https://github.com/symfony/symfony-docs/pull/6614/files#diff-42d49998b5b27b6669bad0e79210dee0R94

after;
https://github.com/symfony/symfony/issues/17201